components/post/posts.jsx

```diff
@@ -46,38 +97,34 @@ function Post({ post }) {
 const PAGE_SIZE = 10;
 
 export function usePostPages({ creatorId } = {}) {
-  return useSWRInfinite((index, previousPageData) => {
-    // reached the end
-    if (previousPageData && previousPageData.posts.length === 0) return null;
+  return useSWRInfinite(
+    (index, previousPageData) => {
+      // reached the end
+      if (previousPageData && previousPageData.posts.length === 0) return null;
 
-    // first page, previousPageData is null
-    if (index === 0) {
-      return `/api/posts?limit=${PAGE_SIZE}${
-        creatorId ? `&by=${creatorId}` : ''
-      }`;
-    }
+      // first page, previousPageData is null
+      if (index === 0) {
+        return `/api/posts?limit=${PAGE_SIZE}${creatorId ? `&by=${creatorId}` : ''}`;
+      }
 
-    // using oldest posts createdAt date as cursor
-    // We want to fetch posts which has a datethat is
-    // before (hence the .getTime() - 1) the last post's createdAt
-    const from = new Date(
-      new Date(
-        previousPageData.posts[previousPageData.posts.length - 1].createdAt,
-      ).getTime() - 1,
-    ).toJSON();
+      // using oldest posts createdAt date as cursor
+      // We want to fetch posts which has a datethat is
+      // before (hence the .getTime() - 1) the last post's createdAt
+      const from = new Date(
+        new Date(previousPageData.posts[previousPageData.posts.length - 1].createdAt).getTime() - 1
+      ).toJSON();
 
-    return `/api/posts?from=${from}&limit=${PAGE_SIZE}${
-      creatorId ? `&by=${creatorId}` : ''
-    }`;
-  }, fetcher, {
-    refreshInterval: 10000, // Refresh every 10 seconds
-  });
+      return `/api/posts?from=${from}&limit=${PAGE_SIZE}${creatorId ? `&by=${creatorId}` : ''}`;
+    },
+    fetcher,
+    {
+      refreshInterval: 10000, // Refresh every 10 seconds
+    }
+  );
 }
```

Comment:
Lots of these changes are stylistic changes (maybe because of some formatting tool on your local), which make it difficult to review.

Is it possible to undo these?

Reply:
Will take care of it.